Pages/Feedback.cshtml

```diff
@@ -35,16 +35,16 @@
                         <div class="contact_form-container">
                             <div>
                                 <div>
-                                    <input type="text" placeholder="Full Name" />
+                                    <input type="text" placeholder="Full Name" name="fullname"/>
                                 </div>
                                 <div>
-                                    <input type="email" placeholder="Email" />
+                                    <input type="email" placeholder="Email" name="email" />
                                 </div>
                                 <div>
-                                    <input type="text" placeholder="Phone Number" />
+                                    <input type="text" placeholder="Phone Number" name="phoneNumber"/>
                                 </div>
                                 <div>
-                                    <input type="text" placeholder="Address" />
+                                    <input type="text" placeholder="Address" name="address"/>
```

Comment:
These were suggested in a previous code review.  When that occurs, first in wins.